packages/benchpress/src/metric/perflog_metric.ts

```diff
@@ -238,13 +238,25 @@ export class PerflogMetric extends Metric {
     events.forEach((event) => {
       const ph = event['ph'];
       const name = event['name'];
-      if (ph === 'B' && name === markName) {
+
+      // Here we are determining if this is the event signaling the start or end of our performance
+      // testing (this is triggered by us calling #timeBegin and #timeEnd).
+      //
+      // Previously, this was done by checking that the event name matched our mark name and that
+      // the phase was either "B" or "E" ("begin" or "end"). However, for some reason now this is
+      // showing up as "-bpstart" and "-bpend" ("benchpress start/end"), which is what one would
+      // actually expect since that is the mark name used in ChromeDriverExtension - see the
+      // #timeBegin and #timeEnd implementations in chrome_driver_extension.ts. We are not sure why
+      // the markName didn't show up with the "-bp(start/end)" suffix before.
```

Comment:
I'd propose adding information about Chrome version update (and a couple minor suggestions):

```suggestion
      // Previously, this was done by checking that the event name matched our mark name and that
      // the phase was either "B" or "E" ("begin" or "end"). However, since Chrome v90 this is
      // showing up as "-bpstart" and "-bpend" ("benchpress start/end"), which is what one would
      // actually expect since that is the mark name used in ChromeDriverExtension - see the
      // #timeBegin and #timeEnd implementations in chrome_driver_extension.ts. For backwards-compatibility
      // with Chrome v89 (and older), we do both checks: the phase-based one ("B" or "E") and event name-based
      // (the "-bp(start/end)" suffix).
```

Reply:
@wagnermaciel FYI, we'll land this PR in the current state (since the feedback was only related to comments) to get targets in g3 back to the "green" state. If you get a chance, could you please update the comment in a followup PR? Thank you.